### DISCUSSION TITLE
Question about Porsche Update with Bose

### DISCUSSION THREAD

**Discussion by ledbette** [Mr-MIBonk/M.I.B._More-Incredible-Bash#242] [Q&A]
I want to flash the MHI2_US_POG11_K5186_1_AIO_MU1476_20230318 onto my 2017 Porsche Cayenne PCM which is currently running MHI2_US_POG11_K3300_MU0809.  

In this Porsche Rennlist post https://rennlist.com/forums/general-porsche-discussions-forum/1242329-pcm4-mib2-mhi2_er_pog11-and-pog24-fw-update-and-patch-navigation-activation.html, there is a general warning that there may be some Bose compatibility issues with flashing new firmware with the Bose system, but doesn't provide any specifics.  They recommend bypassing the Bose link or disabling the Bose-related updates.

Can anyone confirm whether there is a potential coding issue with Bose?  Seems like other forums (like this (https://www.718forum.com/threads/android-auto-i-have-an-oem-like-solution-working.20915/) haven't indicated any issues.

**Comment by harman-f** [comment]
https://github.com/harman-f/MHI2_US_POG11_K5186_1-MU1476-AIO

Using AIO Bose will not be updated.

**Reply by JalilCejin** [reply]
hi harman-f

this AIO is for POG11, if I got POG24 and Bose, is it safe to update directly?

Thanks
